wire-tests/src/jvmJavaTest/proto-java/com/squareup/wire/protos/custom_options/FooBar.java

```diff
@@ -563,15 +563,15 @@ public More redact(More value) {
   }
 
   public enum FooBarBazEnum implements WireEnum {
-    FOO(1, 17, new More.Builder()
+    FOO(1, new More.Builder()
```

Comment:
why does the argument ordering change?

Reply:
Let me check, I probably mixed up some unrelated code.